deployment/gcp/cloudbuild_admin_app.yaml

```diff
@@ -0,0 +1,21 @@
+steps:
+  - name: "gcr.io/cloud-builders/docker"
+    entrypoint: "bash"
+    args:
+      - -c
+      - >
+        docker build \
+          --build-arg NEXT_PUBLIC_BACKEND_URL=https://$$DOMAIN/api \
+          --build-arg NEXT_PUBLIC_GOOGLE_LOGIN_CLIENT_ID=$$NEXT_PUBLIC_GOOGLE_LOGIN_CLIENT_ID \
+          -t ${_DOCKER_REGISTRY_DOMAIN}/$PROJECT_ID/${_RESOURCE_PREFIX}/admin_app:latest \
+          .
+    secretEnv: ["NEXT_PUBLIC_GOOGLE_LOGIN_CLIENT_ID", "DOMAIN"]
```

Comment:
Secret environment variables are accessed by `$$<var>`, substituted variables (which we pass in github actions) are accessed by `$<var>` and must start with `_`